ClosedXML/Excel/CalcEngine/Functions/Tally.cs

```diff
@@ -165,6 +165,15 @@ public double Min()
             return nums.Length == 0 ? 0 : nums.Min();
         }
 
+        public double Fisher()
+        {
+            var nums = NumericValuesInternal();
+            if (nums.Length == 0) throw new ArgumentException("Parameter non numeric.");
```

Comment:
Check the online Excel documentation for the function and throw one of the appropriate ClosedXML exceptions in the `ClosedXML.Excel.CalcEngine.Exceptions` namespace.

Reply:
I fixed this. I don't see this specific exceptions initially.

Reply:
Now I updated this code and I don't use now Tally class.
This is working fine but I don't sure about this:
`double.TryParse(p?[0].Evaluate().ToString(), out double value)`
Is this a good solution for checking if a value is a double? I don't sure. If not please tell me about good way and I will update this check. In this moment I don't good know this codebase. Thanks.